### PR TITLE
add windows boot jdk fetching

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: build-farm/platform-specific-configurations/windows.sh [hunk 1]
```diff
@@ -27,6 +27,21 @@ export OPENJ9_NASM_VERSION=2.13.03
...
+# Any version above 8
+if [ "$JAVA_FEATURE_VERSION" -gt 8 ]; then
+    BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION-1))"
+    BOOT_JDK_VARIABLE="JDK$(echo $BOOT_JDK_VERSION)_BOOT_DIR"
+    if [ ! -d "$(eval echo "\$$BOOT_JDK_VARIABLE")" ]; then
+      export $BOOT_JDK_VARIABLE="$PWD/jdk-$BOOT_JDK_VERSION"
+      if [ ! -d "$(eval echo "\$$BOOT_JDK_VARIABLE/bin")" ]; then
+        wget -q "https://api.adoptopenjdk.net/v2/binary/releases/openjdk${BOOT_JDK_VERSION}?os=windows&release=latest&arch=x64&heap_size=normal&type=jdk&openjdk_impl=hotspot" -O openjdk.zip
+        unzip -q openjdk.zip
+        mv $(eval ls -d jdk-$BOOT_JDK_VERSION*) jdk-$BOOT_JDK_VERSION
```
Comment (johnoliver) [inline build-farm/platform-specific-configurations/windows.sh]: what does the eval do here?

Comment (gdams) [reply build-farm/platform-specific-configurations/windows.sh]: nothing, not sure why it's there aha

**Comment by karianna** [comment]
Shouldn't our infra boxes have had these ansibled in?

**Comment by karianna** [comment]
Shouldn't our infra boxes have had these ansibled in?  Ah, this catches it when it hasn't and is helpful for external folks.  Carry on :-)

**Comment by gdams** [comment]
no we've moved all the other platforms to download from the API.

**Pull request merged**